packages/server/src/fhir/search.ts

```diff
@@ -389,18 +482,20 @@ async function getSearchRevIncludeEntries(
     value = resources.map(getReferenceString).join(',');
   }
 
-  return (
-    await getSearchEntries(repo, {
-      resourceType: revInclude.resourceType as ResourceType,
-      filters: [
-        {
-          code: revInclude.searchParam,
-          operator: Operator.EQUALS,
-          value: value,
-        },
-      ],
-    })
-  ).entry;
+  const searchRequest = {
+    resourceType: revInclude.resourceType as ResourceType,
+    filters: [
+      {
+        code: revInclude.searchParam,
+        operator: Operator.EQUALS,
+        value: value,
+      },
+    ],
+    count: DEFAULT_MAX_SEARCH_COUNT,
```

Comment:
Same as above; previous this search didn't have a count which caused the resulting select query to not have a limit specified anywhere.